masonite/commands/DevCommand.py

```diff
@@ -0,0 +1,39 @@
+"""Dev Command."""
+import os
+import shutil
+import subprocess
+
+from cleo import Command
+
+
+class DevCommand(Command):
+    """
+    Run dev commands
+
+    dev
+        {location=None : Location of file/directory to test or lint}
+        {--t|--test : Run Pytest on file or directory}
+        {--l|--lint : Run Flake8 on file or directory}
+        {--c|--clean : Clean project. Remove .pyc files and __pycache__/ directories}
+    """
+
+    def handle(self):
+
+        # Run pytest
+        if self.option('test'):
+            location = self.argument('location').lower() if self.argument('location') != 'None' else '.'
+            subprocess.call(['python3 -m pytest {location} -v --color=yes'.format(location=location)], shell=True)
+
+        # Run Flake8
+        if self.option('lint'):
+            location = self.argument('location').lower() if self.argument('location') != 'None' else 'masonite/'
+            subprocess.call(['flake8 {location} --ignore=E501,F401,E128,E402,E731,F821,E712,W503'.format(location=location)], shell=True)
+
+        # Clean repo (Works for Windows, MacOS, and Linux)
+        if self.option('clean'):
+            subprocess.call(["python3", "-c", "import pathlib; [p.unlink() for p in pathlib.Path('.').rglob('*.py[co]')]"])
+            subprocess.call(["python3", "-c", "import pathlib; [p.rmdir() for p in pathlib.Path('.').rglob('__pycache__')]"])
+
+        # Command did nothing
+        if not self.option('test') and not self.option('lint') and not self.option('clean'):
+            print('No actions selected')
```

Comment:
get rid of the if statement here on the bottom, use return statements at the end of each if statement and then don't use `print`. Use `self.info` or related commands. Its prettier.

Reply:
Actually, I think there should be the third conditional. The dev can do a combination of all three things. (i.e. `craft dev --test tests/ --clean`) I did not want them to run `craft dev`, and expect for things to happen. Early returns will hinder this command.

Reply:
ok fair point.